packages/core/src/components/table/table-component-basic.stories.tsx

```diff
@@ -127,6 +149,8 @@ export default {
   },
   args: {
     modeVariant: 'Inherit from parent',
+    headerTextAlignment: 'center',
```

Comment:
```suggestion
    headerTextAlignment: 'left',
```

Use left as default.